controllers/api/file/linkAction/AddFileLinkAction.inc.php

```diff
@@ -109,7 +109,7 @@ function _getTextLabels($fileStage) {
 			),
 			SUBMISSION_FILE_PROOF => array(
 				'wizardTitle' => 'submission.upload.proof',
-				'buttonLabel' => 'submission.addFile'
+				'buttonLabel' => 'submission.changeFile'
```

Comment:
This is too general a change. Perhaps a constructor parameter with an `addFile` default that can be changed to `changeFile` in the specific use case?